src/components/map/MapService.js

```diff
@@ -850,7 +897,8 @@ goog.require('ga_urlutils_service');
               return that.getCesiumImageryProviderById(bodId);
             };
           }
-
+          retryOnError(loaderror, loadel, olSource);
+          refreshAfterRetry(loadsuccess, loadel, olSource, olLayer);
```

Comment:
Thinking of external WMS layer, shouldn't we use a general function (in gaMapUtils ? geDefineProperties? ) which register the event on the ol layer in parameter. 

This will make the functionnality more testable too.